src/main/java/com/hermesworld/ais/galapagos/topics/service/impl/MessagesService.java

```diff
@@ -0,0 +1,54 @@
+package com.hermesworld.ais.galapagos.topics.service.impl;
+
+import jakarta.servlet.http.HttpServletRequest;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.context.request.RequestContextHolder;
+import org.springframework.web.context.request.ServletRequestAttributes;
+import org.springframework.web.servlet.support.RequestContextUtils;
+
+import java.text.MessageFormat;
+import java.util.Locale;
+import java.util.MissingResourceException;
+import java.util.ResourceBundle;
+
+@Slf4j
+public class MessagesService {
```

Comment:
Please rename this class to `MessagesServiceImpl` and create an interface `MessagesService` (in package `com.hermesworld.ais.galapagos.messages`), which then is implemented by this class.